### DISCUSSION TITLE
posts/Javascript/%ED%81%B4%EB%A1%9C%EC%A0%80%EC%97%90%20%EB%8C%80%ED%95%98%EC%97%AC

### DISCUSSION THREAD

**Discussion by giscus** [phnml1/my-blog#14] [Announcements]
# posts/Javascript/%ED%81%B4%EB%A1%9C%EC%A0%80%EC%97%90%20%EB%8C%80%ED%95%98%EC%97%AC

자바스크립트의 어렵지만 중요한 클로저의 개념과 활용 이해해보자

http://localhost:3000/posts/Javascript/%ED%81%B4%EB%A1%9C%EC%A0%80%EC%97%90%20%EB%8C%80%ED%95%98%EC%97%AC

**Comment by phnml1** [comment]
ffrffrrf

**Comment by choihooo** [comment]
# 클로저에 대해서 이해가 잘 된 거 같아요
